lib/icinga/downtime.cpp

```diff
@@ -359,7 +359,7 @@ Downtime::Ptr Downtime::AddDowntime(const Checkable::Ptr& checkable, const Strin
 	return downtime;
 }
 
-void Downtime::RemoveDowntime(const String& id, bool includeChildren, bool cancelled, bool expired,
+void Downtime::RemoveDowntime(const String& id, bool includeChildren, bool cancelled, bool force,
```

Comment:
Or! I'd make another bool param not to calculate the hash twice. While on it, those 3 may be worth merging into one enum.